### ISSUE TITLE
Error while obtaining header information

### ISSUE THREAD

**Issue by kazimolmez** [myFMbutler/myFMApiLibrary-for-PHP#10]
### How to fix;

Response.php line 68:
old:
```$this->headers = $headers;```
new:
```$this->headers = array_change_key_case($headers,CASE_LOWER);```

and Response.php line 81,82:
old:
```
if (isset($this->headers[$header])) {
    return $this->headers[$header];
}
```
new:
```
if (isset($this->headers[strtolower($header)])) {
    return $this->headers[strtolower($header)];
}
```

**Comment by JacobTaylor-RCC** [comment]
Good catch -- this is due to http2 changes (requiring lowercase headers). 

We solved this in our fork: https://github.com/rcconsulting/myFMApiLibrary-for-PHP/commit/d4dca9eaff07be16c8aaae11c8e9c889e549b19b

**Comment by lguilbert** [comment]
Hi.

Thanks for the review but: I'm no longer a part of Lesterius anymore; my access is closed. I'm not able to accept/declined this and I don't think this will happen from someone else.

I did the same fix on my own fork : https://github.com/lguilbert/myFMApiLibrary-for-PHP

But you could do your own too.
